handlers.go

```diff
@@ -49,11 +49,17 @@ func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 		httpError(w, r, http.StatusBadRequest, "failed to read request body: %s", err)
 		return
 	}
+
 	if len(body) < 10 {
 		// it's impossible to have a valid request body smaller than 10 bytes
 		httpError(w, r, http.StatusBadRequest, "empty or invalid request request body")
 		return
 	}
+	if len(body) > 104857600 {
+		// the max body size is hardcoded to 100MB. Seriously, what are you trying to sign?
+		httpError(w, r, http.StatusBadRequest, "request exceeds max size of 100MB")
```

Comment:
👍 This seems generous, but it's good to set a limit.

Reply:
We might have to sign larger blobs of data in the future so I'm being extra cautious.